src/V3LinkDot.cpp

```diff
@@ -4945,71 +5154,185 @@ class LinkDotResolveVisitor final : public VNVisitor {
             }
             VL_DO_DANGLING(pushDeletep(cpackagep->unlinkFrBack()), cpackagep);
         }
+        const bool captureEnabled = LinkDotIfaceCapture::enabled();
+        // Single map lookup instead of contains() + find() + getCapturedTypedef()
+        const auto* const captureEntry
+            = captureEnabled ? LinkDotIfaceCapture::find(nodep) : nullptr;
```

Comment:
Check `p` suffix on inserted pointers.  Suggest abbreviate as names getting long.

```suggestion
        const auto* const capEntryp
            = captureEnabled ? LinkDotIfaceCapture::find(nodep) : nullptr;
```

Reply:
Changed this one as well as captureEnabled.